data/json/furniture.json

```diff
@@ -212,20 +212,71 @@
     "bgcolor": "brown",
     "move_cost_mod": -1,
     "coverage": 95,
-    "description": "A few stacked sandbags.",
+    "description": "A sandbag wall.",
     "required_str": -1,
     "flags": [ "NOITEM", "BLOCKSDOOR", "EASY_DECONSTRUCT", "MINEABLE", "BLOCK_WIND" ],
-    "deconstruct": {
-      "items": [ { "item": "bag_canvas", "count": 4 }, { "item": "material_sand", "charges": 40 } ],
-      "furn_set": "f_sandbag_half"
-    },
+    "deconstruct": { "items": [ { "item": "sandbag", "count": 20 } ], "furn_set": "f_sandbag_half" },
     "bash": {
       "str_min": 24,
       "str_max": 80,
       "sound": "rrrip!",
       "sound_fail": "whump.",
       "furn_set": "f_sandbag_half",
-      "items": [ { "item": "bag_canvas", "count": [ 1, 4 ] }, { "item": "material_sand", "charges": [ 10, 40 ] } ]
+      "items": [ { "item": "bag_canvas", "count": [ 15, 20 ] }, { "item": "material_sand", "charges": [ 1000, 1200 ] } ]
+    }
+  },
+  {
+    "type": "furniture",
+    "id": "f_earthbag_half",
+    "name": "earthbag barricade",
+    "symbol": "#",
+    "looks_like": "f_sandbag_half",
+    "bgcolor": "brown",
+    "description": "An earthbag barricade, typically used for blocking bullets.",
+    "move_cost_mod": -1,
+    "coverage": 60,
+    "required_str": -1,
+    "flags": [
+      "CLIMB_SIMPLE",
+      "TRANSPARENT",
+      "MOUNTABLE",
+      "BLOCKSDOOR",
+      "SHORT",
+      "EASY_DECONSTRUCT",
+      "THIN_OBSTACLE",
+      "CLIMBABLE",
+      "PERMEABLE"
+    ],
+    "examine_action": "chainfence",
+    "deconstruct": { "items": [ { "item": "sandbag", "count": 16 } ] },
```

Comment:
Deconstruct and bash results for this one (and earthbag wall) still mention sand and sandbags. Likely unintended.

Reply:
You're right, that's an editing oversight.